openupgradelib/openupgrade.py

```diff
@@ -376,6 +377,108 @@ def copy_columns(cr, column_spec):
             })
 
 
+def copy_fields_multilang(cr, destination_model, destination_table,
+                          destination_columns, relation_column,
+                          source_model=None, source_table=None,
+                          source_columns=None):
+    """Copy field contents including translations.
+
+    :param str destination_model:
+        Name of the destination model where the data will be copied to.
+
+    :param str destination_table:
+        Name of the destination table where the data will be copied to.
+        It must be ``env[destination_model]._table``.
+
+    :param list destination_columns:
+        List of column names in the :param:`destination_table` that will
+        receive the copied data.
+
+    :param str relation_column:
+        Name of a column in :param:`destination_table` which points to IDs in
+        :param:`source_table`. An ``INNER JOIN`` will be done to update the
+        destination records with their corresponding source records only.
+        Records where this column ``NULL`` will be skipped.
+
+    :param str source_model:
+        Name of the source model where the data will be copied from.
+        If empty, it will default to :param:`destination_table`.
+
+    :param str source_table:
+        Name of the source table where the data will be copied from.
+        If empty, it will default to :param:`destination_table`.
+        It must be ``env[source_model]._table``.
+
+    :param list source_columns:
+        List of column names in the :param:`source_table` that will
+        provide the copied data.
```

Comment:
Missed "If empty, it will default to :param:`destination_columns`."

Reply:
Fixed, thanks.